internal/nodes/avi_vrf_translator.go

```diff
@@ -107,19 +113,32 @@ func (o *AviObjectGraph) BuildVRFGraph(key, vrfName, nodeName string, deleteFlag
 		startIndex := nodeStaticRouteDetails.StartIndex
 		count := nodeStaticRouteDetails.Count
 		var staticRouteCopy []*models.StaticRoute
+
+		countToSubstract := aviVrfNode.NodeStaticRoutes[nodeName].Count
+		updateNodeStaticRoutes(aviVrfNode, deleteFlag, nodeName, 0, -countToSubstract)
 		staticRouteCopy = append(staticRouteCopy, aviVrfNode.StaticRoutes[:startIndex]...)
 		staticRouteCopy = append(staticRouteCopy, aviVrfNode.StaticRoutes[startIndex+count:]...)
 		aviVrfNode.StaticRoutes = staticRouteCopy
-		countToSubstract := aviVrfNode.NodeStaticRoutes[nodeName].Count
-		updateNodeStaticRoutes(aviVrfNode, deleteFlag, nodeName, 0, -countToSubstract)
 		delete(aviVrfNode.NodeStaticRoutes, nodeName)
+		//TODO: Optimize
+		for i := len(aviVrfNode.NodeIds); i > len(aviVrfNode.StaticRoutes); i-- {
```

Comment:
this seems to me an optimized code since we will be dealing with at most 1 route with either ipv4 or ipv6 and 2 routes when we support dual stack.

Reply:
addressed